mock/environment/__init__.py

```diff
@@ -16,34 +17,49 @@
 class Env:
    
     def __init__(self):
+        """
+        Stores weather data into a dictionary 
+        that can be indexed by site and date.
+        """
         self.site_data = {}
         self.site_data_by_night = {}
-        
+
         for site in Site:
             site_lc = site.name.lower()
             input_filename = Env.data_file_path(f'{site_lc}_weather_data.pickle.bz2')
-            # count = 0
 
             if(os.path.exists(input_filename)):
                 # logging.info(f'Reading {input_filename}')
                 with bz2.open(input_filename) as input_file:
                         input_data = pd.read_pickle(input_file)
                         self.site_data[site_lc] = input_data 
                         # logging.info(site_lc)
-                        # logging.info(self.site_data[site_lc].iloc[0:2])
+                        # logging.info(self.site_data[site_lc])
                         # logging.info(f'\t{len(self.site_data[site_lc].columns)} columns, {len(self.site_data[site_lc])} rows')
-                # self.site_data_by_night[site_lc] = {}
-                # for night in self.site_data[site_lc]["Time_Stamp_UTC"]:
-                #     night_date = night.date()
-                #     night_date = night_date.strftime('%Y-%m-%d') 
-                #    # logging.info(night_date)
-                #     self.site_data_by_night[site_lc][night_date] = self.site_data[site_lc].iloc[count]
-                #     count += 1
-                #     print(self.site_data_by_night[site_lc][night_date])
-            else:
+
+                self.site_data_by_night[site_lc] = {}
+                local_site_data = self.site_data[site_lc].iterrows()
+                for _index, night in local_site_data:
```

Comment:
I thought the front index is used to show an unused variable

Reply:
Oh, yeah, the pattern `_variable` is used to indicate that variables should be treated as private in a class, but at the same time, if you're not using a value, you should just call it `_` with nothing after it.

That's why most of the variables in classes are usually labeled:
```
self._variable
```
so that they're only accessible through methods or used in internal calculations.